yaml-tests/src/test/java/YamlIntegrationTests.java

```diff
@@ -297,7 +297,6 @@ public void transactionalCallsTest(YamlTest.Runner runner) throws Exception {
     }
 
     @TestTemplate
-    @MaintainYamlTestConfig(YamlTestConfigFilters.CORRECT_EXPLAIN_AND_METRICS)
```

Comment:
Good catch! I thought there was an issue somewhere to make this a PRB failure, but I can't actually find it